### DISCUSSION TITLE
Does the Mastodon iOS app support writing right-to-left Arabic posts?

### DISCUSSION THREAD

**Discussion by Diwannist** [mastodon/mastodon#32812] [Q&A] [answered]
![IMG_7394](https://github.com/user-attachments/assets/9d199ab7-a1da-4043-9300-8818eb2c4690)

**Comment by Diwannist** [comment, marked as answer]
It supports right-to-left only when the post is published, not while it’s being edited.

**Reply by trwnh** [reply]
Hello, if you have a problem with the Mastodon iOS application, the correct place to go to would be https://github.com/mastodon/mastodon-ios

**Reply by Diwannist** [reply]
Okay, thanks.
